client-java/controller/src/main/java/org/evomaster/client/java/controller/internal/SutController.java

```diff
@@ -403,7 +408,7 @@ private void computeSQLHeuristics(ExtraHeuristicsDto dto, List<AdditionalInfo> a
             accessedTables.addAll(sqlExecutionsDto.deletedData);
             accessedTables.addAll(sqlExecutionsDto.insertedData.keySet());
             //accessedTables.addAll(executionDto.queriedData.keySet());
-            accessedTables.addAll(sqlExecutionsDto.insertedData.keySet());
+//            accessedTables.addAll(sqlExecutionsDto.insertedData.keySet());
```

Comment:
why this? add comments

Reply:
because this line of code (line 411) seems duplicated, ie, the `insertedData` has been added at line 409.